### DISCUSSION TITLE
Scoreboard teams not shown when player out of render distance view?

### DISCUSSION THREAD

**Discussion by hugovdev** [Minestom/Minestom#1535] [Help]
https://streamable.com/gb363s
As we can see in that video when a player gets to far from another, the name becomes white in the tab.
A work around is also doing player#setDisplayName but that changes the order of the players in the tab, as teams are not taken into account.

**Comment by matserix** [comment]
How did you create your scoreboard? I can't reproduce your problem

**Reply by hugovdev** [reply]
We found out on the discord that if you do player#setTeam(team) this happens. For some reason if there is only one person on the team the player belongs to it removes the team packet from the receiver.
https://github.com/Minestom/Minestom/blob/1a013728fd54dde4e0ab5749a96307e7483d69ca/src/main/java/net/minestom/server/entity/Player.java#L531
